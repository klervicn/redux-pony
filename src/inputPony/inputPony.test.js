import React from 'react';
import InputPony from './inputPony';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<InputPony onClick={jest.fn} />).toJSON();

  expect(tree).toMatchSnapshot();
});
