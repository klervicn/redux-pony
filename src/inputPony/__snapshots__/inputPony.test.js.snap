// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      type="text"
      value=""
    />
    <button
      type="submit"
    >
      Add
    </button>
  </form>
</div>
`;
